api/go.sum

```diff
@@ -234,6 +235,18 @@ github.com/openstack-k8s-operators/openstack-ansibleee-operator/api v0.0.0-20230
 github.com/openstack-k8s-operators/openstack-ansibleee-operator/api v0.0.0-20230620085730-fddf65f65f88/go.mod h1:e4hw70sdvXG8anfBSGq5VMq11h52aN/TI3HYxjPyDGE=
 github.com/openstack-k8s-operators/openstack-baremetal-operator/api v0.0.0-20230626134559-c08170fb5cff h1:ui7RJZdA9SedC1joPNfvkNQ/SeF5LSJmlTyfJhzaDxY=
 github.com/openstack-k8s-operators/openstack-baremetal-operator/api v0.0.0-20230626134559-c08170fb5cff/go.mod h1:JTCJLOYuhAr5GZiPzctpcOTh3IvqJn6RvvbiTMRoUQ4=
+=======
```

Comment:
Can you please squash the commits as they would be committed as separate ones. Having merge conflicts in  commit like this is not good.